src/Containers/Lobby/index.tsx

```diff
@@ -22,19 +27,25 @@ const Lobby = () => {
                 </Button>
               </Col>
               <Col span={12}>
-                <StartButton disabled={playerCount < 2}>Start</StartButton>
+                <StartButton disabled={players.length < 2 || selfID !== owner.id}>Start</StartButton>
```

Comment:
StartButton should send `POST /api/v1/game/start/ `. This has to be done now or just remember about it in the task with game session view